### DISCUSSION TITLE
simulate datasets from prior data (model fit)

### DISCUSSION THREAD

**Discussion by Minshi-Lan** [nlmixrdevelopment/nlmixr#636] [General]
Hi everyone, I want to simulate datasets with the model fit, when I try **sim.2cpt <- simulate(fit, events = ev, nSub = 500)** , as the tutorial paper did, error occurred. It reads that provided multiple subjects dataset (nSub = 12) and trying to simulate different number of Subjects (nSub = 500). 

Can I simulate  dataset with different number of Subjects from prior dataset ?

Looking forward to any reply.

BW

**Comment by mattfidler** [comment]
Yes, you can.  Try using `rxSolve()` instead.  It is more likely to work.
